### PR TITLE
English: CAPP Agile Learning page

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Translates CAPP Agile Learning page

### CODE REVIEW

**Comment by fatboypunk** [comment]
needs a rebase

**Comment by RonaldDijkstra** [comment]
Still need to translate the images, see issue #618

**Comment by RonaldDijkstra** [comment]
There's only 1 image left todo for this page, which I cannot do before this question is answered: #648. But i'd like a review first. 👍

File: locales/en.yml [hunk 1]
```diff
@@ -54,9 +54,77 @@ en:
...
+    why:
+      heading: "Why choose CAPP Agile Learning?"
+      make_share:
+        heading: "Create & share"
+        copy: >
+          Support your staff with instructions, useful video tutorials or graphics, and the tips and tricks from the experts in your organisation. Our platform is a sleek and professional medium for sharing knowledge effortlessly with your employees.
+      on_demand:
+        heading: "On-demand learning"
+        copy: >
+          Make optimal use of your organisation's expertise - wherever, whenever. CAPP Agile Learning is available for all devices. Use the QR codes to obtain the right information just in time. No more time-consuming searches in cluttered document systems.
```
Comment (sn3p) [inline locales/en.yml]: > ... obtain the **correct** information ~~just in time~~.

File: locales/en.yml [hunk 2]
```diff
@@ -54,9 +54,77 @@ en:
...
+
+          Use the Shared Spaces functionality to make information available without having to log on to the platform.
+
+          This ensures that every user receives the information required for optimal performance exactly when they need it, in other words: just in time.
+        img: screenshots/capp-agile-qr-code_nl.png
+        img_alt: "On demand with QR-codes"
+      demand_driven:
+        heading: "Demand-driven"
+        copy: |
+          It is impossible to predict exactly which expertise will be required within an organisation. A lack of knowledge is often discovered (too) late because the need for this expertise is hidden. CAPP Agile Learning solves this problem by starting a dialogue with .
```
Comment (sn3p) [inline locales/en.yml]: > by starting a dialogue **with** .

With what?

File: locales/en.yml (hunk 1 above)
Comment (sn3p) [inline locales/en.yml]: Mhm or is it **right**?

**Pull request merged**